apis/v1alpha2/referencegrant_types.go

```diff
@@ -35,21 +39,15 @@ import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 // All cross-namespace references in Gateway API (with the exception of cross-namespace
 // Gateway-route attachment) require a ReferenceGrant.
 //
-// Support: Core
+// ReferenceGrant is a form of runtime verification. Implementations that support
+// ReferenceGrant MUST respond to removal of a grant by revoking the access that
+// grant allowed.
 //
-type ReferenceGrant struct {
-	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
-
-	// Spec defines the desired state of ReferenceGrant.
-	Spec ReferenceGrantSpec `json:"spec,omitempty"`
-
-	// Note that `Status` sub-resource has been excluded at the
-	// moment as it was difficult to work out the design.
-	// `Status` sub-resource may be added in future.
-}
+// Support: Core
+type ReferenceGrant v1beta1.ReferenceGrant
 
 // +kubebuilder:object:root=true
+
```

Comment:
nit.

Reply:
Removed in 7a4f57fba2af51813da1a88ad89a9266a765b477 of #1569